### PR TITLE
Removing unnecessary opening of a fake volume due to old configuration related issues

### PR DESCRIPTION
**Pull request by mynktl**
Changes : 
Disabling volume file related operations if replica feature is enabled

##### _Note: from istgt.conf, volume file is not yet removed. I will take care of this in my next PR (DE35)._

Signed-off-by: mayank <mayank.patel@cloudbyte.com>

### CODE REVIEW

File: src/istgt_lu_disk.c [hunk 1]
```diff
@@ -6082,6 +6105,7 @@ istgt_lu_disk_lbwrite_ats(ISTGT_LU_DISK *spec, CONN_Ptr conn, ISTGT_LU_CMD_Ptr l
 	return 0;
 }
 
+#ifndef	REPLICATION
```
Comment (vishnuitta) [inline src/istgt_lu_disk.c]: we need lbsync code

File: src/istgt_lu.c [hunk 2]
```diff
@@ -4217,11 +4226,13 @@ maintenance_io_worker(void *arg)
 				} else if (spec->ex_state == ISTGT_LUN_OPEN_PENDING) {
 						do_open = 1;
 				}
+#ifndef	REPLICATION
```
Comment (vishnuitta) [inline src/istgt_lu.c]: 'unlock' need to be out of this '#ifndef'

File: src/istgt_lu_disk.c [hunk 3]
```diff
@@ -10220,6 +10254,7 @@ istgt_lu_disk_execute(CONN_Ptr conn, ISTGT_LU_CMD_Ptr lu_cmd)
 			    "c#%d SYNCHRONIZE_CACHE_10(lba %"PRIu64
 			    ", len %u blocks)\n",
 			    conn->id, lba, len);
+#ifndef	REPLICATION
```
Comment (vishnuitta) [inline src/istgt_lu_disk.c]: this is required

File: src/istgt_lu_disk.c (hunk 1 above)
Comment (mynktl) [reply src/istgt_lu_disk.c]: ok

File: src/istgt_lu.c (hunk 2 above)
Comment (mynktl) [reply src/istgt_lu.c]: ok

File: src/istgt_lu_disk.c (hunk 3 above)
Comment (mynktl) [reply src/istgt_lu_disk.c]: ok

**Review by vishnuitta — APPROVED** [review]
changes are good

**Pull request merged**